### ISSUE TITLE
equations numbering inconsistent 

### ISSUE THREAD

**Issue by ldXiao** [plastex/plastex#268]
For simple input
```tex
Given the action and lagrangian density of free boson (real number, which correctly desctibe all free bosons except W and Z bosons)
\begin{equation}\label{eq:aa}
\begin{aligned}
    S[\varphi] &= \int d\mathbf{x}dt \mathcal{L}(\varphi, \dot{\varphi}, \nabla \varphi)\\
    \mathcal{L} &= \frac{1}{2}
    \left\{
      \frac{1}{c^2}\dot{\varphi}^2 -(\nabla \varphi)^2 - m^2 \varphi^2
    \right\}
  \end{aligned}
\end{equation}
consider the discrete case on a 1D lattice($\varphi_n= a* n$), like a chain of oscilators
\begin{equation}
L = \sum^{N-1}_0 \frac{1} a
\left \{
  \dot{\varphi_n}^2 -\frac{1}{a^2}\left(\varphi_{n+1}-\varphi_n\right)^2 - m^2 \varphi_n^2
\right\}
\end{equation}
at the continuous limit $a\rightarrow 0$, it amounts to the quantity $\int \mathcal{L}(\mathbf{x}, t) d\mathbf{x}$
.
```
The generated html file [gives](url)
<img width="681" alt="Screen Shot 2022-01-03 at 8 54 52 AM" src="https://user-images.githubusercontent.com/32606147/147939015-deb8be43-9c22-45c7-9fab-539a6540990a.png">

I have also tried the environment `align` I find the counter add to equation counter, but the numbers do not show  up for `align`.
I  think is is casued by the `align/ aligned` environment, somehow their conuter mix up with the counter of equations.

**Comment by PatrickMassot** [comment]
Thanks for this report. I hope I fixed the `aligned` issue in b441bfea0316458c720ac8f770caaffc920bb2bf, could you confirm? 

The issue with `align` is much harder. This environment is currently considered as a normal math environment to be handled entirely by mathjax. mathjax has some way of numbering stuff, but I don't see a way to mix it with plastex numbering, and mathjax itself cannot handle all the numbering of a complex document. We would need to work on the align environment from within plastex to manually insert `\tag` commands inside the code given to mathjax. This is possible in principle but not high priority for me so it will probably happen in a very long time if nobody helps here. In the mean time you can mannually use `\tag`, although it's a bit sad.

**Comment by ldXiao** [comment]
Hi Patrick, thanks for the quick reply. I have pulled from latest master branch and I can confirm `aligned` issue has been solved, which is quite enough for me.
